CIP-1694/README.md

```diff
@@ -0,0 +1,1163 @@
+(via the **no confidence** action) if they overstep this boundary.
+Said differently, there is a social contract between the constitutional committee and the actors of the network.
+Although the constitutional committee could reject certain governance actions (by voting 'No' on them),
+they should only do so when those governance actions are in conflict with the Constitution.
+
+For example, if we consider the hypothetical Constitution rule "The Cardano network must always be able to produce new blocks",
+then a governance action that would reduce the maximum block size to `0` would be, in effect,
+unconstitutional and so might not be ratified by the constitutional committee.  The rule does
+not, however, specify the smallest acceptable maximum block size, so the constitutional committee would need to determine this number
+and vote accordingly.
+
+#### State of no-confidence
+
+The constitutional committee is considered to be in one of the following two states at all times:
+
+1. a normal state (i.e. a state of confidence)
+2. a state of no-confidence
+
+In a _state of no-confidence_, the current committee is no longer able to participate in governance actions
+and must be replaced before any governance actions can be ratified (see below).
+Any outstanding governance actions are dropped immediately after the protocol enters a state of no-confidence,
+and will not be enacted.
+
+#### Constitutional committee keys
+
+The constitutional committee will use a hot and cold key setup, similar to the existing "genesis delegation certificate" mechanism.
+
+#### Replacing the constitutional committee
+
+The constitutional committee can be replaced in one of two ways:
+
+* When in a **normal** state (i.e. a state of **confidence**), the committee can be replaced via a
+specific governance action ("New constitional committee", described below) that requires the approval of both the **current constitutional committee** and the **DReps**.
+
+* When in a state of **no-confidence**, the committee can also be replaced via the same governance action (action 2 below),
+but this instead requires the approval of both the **SPOs** and the **DReps**.
+
+The new constitutional committee could, in principle, be identical to or partially overlap the outgoing committee as long as the action is properly ratified.
+This might happen, for example, if the electorate has collective confidence in all or part of the committee and wishes to extend its term of office.
+
+
+#### Size of the constitutional committee
+
+Unlike the Shelley governance design, the size of the constitutional committee is not fixed and can be any nonnegative number.
+It may be changed whenever a new committee is elected ("New constitutional committee and/or quorum size").
+Likewise, the committee _quorum_ (the number of committee `Yes` votes that are required to ratify governance actions) is not fixed and
```

Comment:
When the number of 'Yes' Votes from the Constitutional committee reaches the committee quorum, the Governance action is ratified. On the other hand, I can only pretend to know what will happen when certain members of the committee vote 'No' and/or 'Abstain' and the quorum is not reached ('dropped' or nothing til 'expired'). The scenario looks worse when they all vote abstain. (Is the governance action 'Dropped' immediately or does it remain latent despite the measurable fact that it will never be ratified) In short, regardless of these answers, I believe that it would be important to specify it in the CIP -1694 in detail (in accordance of course with what you currently know about the thing,). Thank you =D

Reply:
My reading of it:
 - dReps are allowed to change their vote, so I had assumed CC members could also change their vote
 - Thus, voting "Abstain" doesn't guarantee it won't pass; it could be a holding pattern until some CC members change their vote
 - Thus, it would remain active until the normal expiration (I think 5-10 epochs) killed it.

Reply:
So I recap.
- The Committee quorum not reached = remain active until the normal expiration allowing them to change their votes
- Votes to 'Abstain' which reaches the committee quorum = remain active until the normal expiration to allow them to change their votes (or not)
- Votes to 'No' which reaches the committee quorum = the Governance action 'Dropped'
- Votes to 'YES' which reaches the committee quorum = Governance action is 'Ratified' if the Dreps (and SPO, depending on the action) also reaches their Threshold.
If I'm not mistaken, this would be a nice addition to the CIP.

Reply:
> I believe that it would be important to specify it in the CIP -1694 in detail

This is a great observation @Hornan7 , thank you for bringing it up. We can definitely make this more clear. Currently, as written, there is no semantic difference between `No` and `Abstain`. Which means that if a governance action were to accumulate too many `No`/`Abstain` votes, it would just eventually expire. The better behavior, we now believe, is for `Abstain` to have the same semantics that it does when the DReps use it, namely subtracting from the denominator. But, for this to make since, the quorum then needs to be fractional instead of a whole number. For example, if you had a committee of size 13 and a quorum of >= 2/3, then 9 `Yes`'s would meet the ratification threshold, as would 8 `Yes`'s and 1 `Abstain`. We were planning on adding this one we talked it over with the scientists (credit to @WhatisRT for noticing it).